CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Added a toggle button to let users decide whether or not to keep the previously generated moment images ([#2054](https://github.com/CARTAvis/carta-frontend/issues/2054)).
 * Added settings in the image view settings widget for panning and zooming the images ([#1176](https://github.com/CARTAvis/carta-frontend/issues/1176)).
 ### Fixed
+* Fixed the issue of annoying text input fields in the vector rendering dialog ([#1906](https://github.com/CARTAvis/carta-frontend/issues/1906)).
```

Comment:
@I-Chenn I think we could remove "in the vector rendering dialog" because we fixed all the inputs. Please also resolve the merge conflict, thanks.

Reply:
Done in ac99a21.